packages/sanity/test/cli/schema/mocks/schemaStoreMocks.ts

```diff
@@ -80,47 +89,95 @@ export function createMockCliOutputter() {
   }
 }
 
+export function getMockStoreKey({
+  projectId,
+  dataset,
+}: {projectId: string; dataset: string} | ClientConfig) {
+  return `${projectId}-${dataset}`
```

Comment:
this will cause `undefined-undefined` if ClientConfig without projectId and dataset is passed. Should it be required up-front?

Reply:
Its a mock store, for testing, i think its fine. WTB fake sanity client that is actually fully featured.